db/db_bloom_filter_test.cc

```diff
@@ -16,6 +17,14 @@ namespace ROCKSDB_NAMESPACE {
 
 namespace {
 using BFP = BloomFilterPolicy;
+
```

Comment:
So the change in db/db_bloom_filter_test.cc  is not related to the main purpose of this PR right? If so, I prefer not to make these change in this file.